components/Marketing/Cards.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from 'react'
+import { useSprings, animated, interpolate } from 'react-spring'
+import { useGesture } from 'react-use-gesture'
+import { css } from 'glamor'
+import {
+  Editorial,
+  FigureImage,
+  mediaQueries
+} from '@project-r/styleguide'
+import { negativeColors } from '../Frame/constants'
+import { t } from '../../lib/withT'
+
+const ASSETS_URL = 'https://cdn.republik.space/s3/republik-assets/assets/marketing/'
+const MAX_WIDTH = 800
+
+// Inspired by https://codesandbox.io/s/j0y0vpz59
+
+const styles = {
+  root: css({
+    background: negativeColors.primaryBg,
+    position: 'relative',
+    overflow: 'hidden',
+    width: '100%',
+    // height: '100%',
+    height: '100vh',
+    maxHeight: '600px',
+    [mediaQueries.mUp]: {
+      maxHeight: '1000px'
+    },
+    textAlign: 'center',
+    '& > div': {
+      position: 'absolute',
+      width: '100%',
+      height: '100%',
+      willChange: 'transform',
+      display: 'flex',
+      alignItems: 'flex-start',
+      justifyContent: 'center'
+    },
+    '& > div > div': {
+      cursor: 'grab',
+      '&:active': {
+        cursor: 'grabbing'
+      },
+      backgroundColor: '#fff',
+      backgroundSize: 'auto 85%',
+      backgroundRepeat: 'no-repeat',
+      backgroundPosition: 'center center',
+      width: '85vw',
+      // height: '60vh',
+      maxHeight: '600px',
+      [mediaQueries.mUp]: {
+        maxHeight: '1000px'
+      },
+      maxWidth: `${MAX_WIDTH / 2}px`, // hd
+      willChange: 'transform',
+      borderRadius: '5px',
+      boxShadow: '0 12px 50px -10px rgba(0, 0, 0, 0.4), 0 10px 10px -10px rgba(0, 0, 0, 0.1)',
+      overflow: 'hidden',
+      padding: '20px 15px 15px 15px'
+    },
+    '& *': {
+      userSelect: 'none'
+    }
+  }),
+  title: css({
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    top: '20px',
+    textAlign: 'center',
+    color: negativeColors.text,
+    [mediaQueries.mUp]: {
+      top: '30px'
+    }
+  })
+}
+
+const tt = key => t(`marketing/v2/cards/${key}`)
+
+const cards = [
+  // In reverse order (last is stacked on top).
+  {
+    title: tt('11/title'),
+    subtitle: tt('11/subtitle'),
+    image: `${ASSETS_URL}international.jpg?resize=${MAX_WIDTH}x`
```

Comment:
```suggestion
    image: `${ASSETS_URL}international.jpg?size=3500x2333&resize=${MAX_WIDTH}x`
```
add sizes to the urls for FigureImage to lazy load render aspect ratio placeholder until loaded.